app/models/search.rb

```diff
@@ -41,11 +41,11 @@ def q=(term)
   end
 
   def countries
-    GeographicalArea.all.compact
+    @countries ||= GeographicalArea.all.compact
```

Comment:
I don't remember countries ever coming back with nil entries :thinking: . I'm not convinced it's actually possible from a jsonapi relationship/include serializer perspective. Probably the compact that you didn't add is superfluous.